assets/elm/main.elm

```diff
@@ -699,6 +743,21 @@ viewGame model =
                         []
                     else
                         [ votesView model ]
+
+                Tricky ->
+                    if isTrickster model then
+                        [ button myButtonModifiers [ onClick GuessTopic ] [ text "Guess Topic" ] ]
+                    else
+                        []
+
+                Check ->
+                    if isGameMaster model then
+                        [ text "Was the trickster's guess correct?"
+                        , button myButtonModifiers [ onClick <| Validate <| True ] [ text "Yes" ]
+                        , button myButtonModifiers [ onClick <| Validate <| False ] [ text "No" ]
```

Comment:
These can just be `onClick <| Validate True` or `onClick <| Validate False`. The 2nd back pipe isn't needed